CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 - [#2265](https://github.com/thanos-io/thanos/pull/2265) Compactor: Add `--wait-interval` to specify compaction wait interval between consecutive compact runs when `--wait` enabled.
 
+### Changed
+
+- [#2090](https://github.com/thanos-io/thanos/issues/2090) *breaking* Downsample: `Downsample` has moved as `thanos bucket` sub-command, can not call by `thanos downsample` any more.
```

Comment:
I think I would rearrange this sentence a bit:

Downsample command: the `downsample` command has moved as under the `thanos bucket` sub-command, and cannot be called via `thanos downsample` any more.